de.properties.tmpl

```diff
@@ -124,7 +124,10 @@ org.iplantc.services.tools = {{ key (printf "%s/terrain/base" $base) }}/tools
 org.iplantc.services.admin.workshop = {{ key (printf "%s/terrain/base" $base) }}/admin/groups/workshop
 
 # Path List File Identifier
-org.iplantc.pathList.fileIdentifier = {{ key (printf "%s/de/file-identifier" $base) }}
+org.iplantc.htpathList.fileIdentifier = {{ key (printf "%s/data-info/ht-path-list-file-identifier" $base) }}
+org.iplantc.multiinputList.fileIdentifier = {{ key (printf "%s/data-info/multi-input-path-list-file-identifier" $base) }}
```

Comment:
Minor thing, but it'd be nice to have these 2 identifier keys have consistent camelCase